apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -188,6 +195,7 @@ export class ApplicationController {
     @CurrentUser() user: User,
     @Query('typeId') typeId?: string,
     @Query('status') status?: string,
+    @Query('scopeCheck') scopeCheck?: boolean,
```

Comment:
Should we permit the client to determine whether a scope check should occur? Is it possible for a user to elevate their access privileges by modifying the query?

Reply:
I think its ok as its just filtering out unneeded data, the client can already see his scopes if he wishes to.
But @eirikurn would probably be the best to determine if its a bigger issue.